缺陷: sealos安装kuboard报错

**哪个命令或者组件**
sealos install --pkg-url kuboard.tar 
kuboard版本为v2.0.5.1版本
**描述这个缺陷**
无法安装报错提示如下：
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x18baa48]

goroutine 1 [running]:
github.com/fanux/sealos/pkg/appmanager.InstallApp(0xc0004d7d00, 0x0, 0x0, 0xc0001ac8d0, 0xc0004d7d30)
	/home/runner/work/sealos/sealos/pkg/appmanager/installapp.go:40 +0x148
github.com/fanux/sealos/cmd.glob..func9(0x3372ce0, 0xc000494a80, 0x0, 0x2)
	/home/runner/work/sealos/sealos/cmd/install.go:50 +0x102
github.com/spf13/cobra.(*Command).execute(0x3372ce0, 0xc000494a60, 0x2, 0x2, 0x3372ce0, 0xc000494a60)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x33734c0, 0x442faa, 0x32fb8e0, 0xc0000ec778)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/fanux/sealos/cmd.Execute()
	/home/runner/work/sealos/sealos/cmd/root.go:50 +0x31
main.main()
	/home/runner/work/sealos/sealos/main.go:22 +0x20
```
**重现步骤（如果有需要可以附加图片）**
sealos install --pkg-url kuboard.tar 
...


**测试结果**
其他应用可成功安装

**运行环境**

- Docker版本:   19.03
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:30:33Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:23:04Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
```
- sealos版本: 
```
sealos version v0.0.0-next, build 113605b go1.14.13, Build Time : 2020-12-21T15:49:36Z
```
- 操作系统:  `Linux master1 3.10.0-1127.el7.x86_64 #1 SMP Tue Mar 31 23:36:51 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux  
CentOS7.8`



cpu cores	: 14

- 运行模式:  物理机 （内存 32g, cpu 14核 ,存储 1t）
- 集群规模:   1master 2node

--- a/pkg/appmanager/installapp.go
+++ b/pkg/appmanager/installapp.go
@@ -36,7 +36,11 @@ func InstallApp(flag *InstallFlags, cfgFile string) error {
 		os.Exit(0)
 	}
 
-	pkgConfig, _ := LoadAppConfig(flag.PkgURL, flag.Config)
+	pkgConfig, err := LoadAppConfig(flag.PkgURL, flag.Config)
+	if err != nil {
+		logger.Error("Load App config from tarball err: ", err)
+		os.Exit(0)
+	}
 	pkgConfig.URL = flag.PkgURL
 	pkgConfig.Name = nameFromUrl(flag.PkgURL)
 	pkgConfig.Workdir = flag.WorkDir

